packages/portfolio-contract/test/supports.ts

```diff
@@ -126,61 +130,13 @@ export const axelarCCTPConfig = {
     namespace: 'eip155',
     reference: '250',
   },
-  binance: {
+  Binance: {
     namespace: 'eip155',
     reference: '56',
   },
 };
 
-/**
- * Testnet chains only.
- *
- * Sourced from:
- *
- * - https://developers.circle.com/stablecoins/supported-domains
- * - https://chainlist.org/
- * - https://docs.simplehash.com/reference/supported-chains-testnets (accessed on
- *   4 July 2025)
- *
- * @satisfies {Record<string, import('./orchestration-api').BaseChainInfo>}
- */
-const axelarCCTPConfigTestnet = {
-  'ethereum-sepolia': {
-    namespace: 'eip155',
-    reference: '11155111',
-    cctpDestinationDomain: 0,
-  },
-  Avalanche: {
-    namespace: 'eip155',
-    reference: '43113',
-    cctpDestinationDomain: 1,
-  },
-  'optimism-sepolia': {
-    namespace: 'eip155',
-    reference: '11155420',
-    cctpDestinationDomain: 2,
-  },
-  'arbitrum-sepolia': {
-    namespace: 'eip155',
-    reference: '421614',
-    cctpDestinationDomain: 3,
-  },
-  'polygon-sepolia': {
-    namespace: 'eip155',
-    reference: '80002',
-    cctpDestinationDomain: 7,
-  },
-  Fantom: {
-    namespace: 'eip155',
-    reference: '4002', // XXX: confirm this ID
-  },
-  binance: {
-    namespace: 'eip155',
-    reference: '97',
-  },
-};
-
-const ccptConfig = { ...axelarCCTPConfig, ...axelarCCTPConfigTestnet };
+const ccptConfig = { ...axelarCCTPConfig };
```

Comment:
any particular reason for the spread / copy?

```suggestion
const ccptConfig = axelarCCTPConfig;
```

I'm curious why the alias, in fact.

Reply:
Thanks for pointing it out! There’s no specific reason for using the spread operator and the alias at this point. I think when I first set this up a while ago, I was running into some mutability-related issues. I didn’t revisit it afterward and just carried it forward without much thought.